### PR TITLE
Change logo width on sliders

### PR DESCRIPTION
**Pull request by hectoruch**
Change LDF logo width when change the slider.

Task=>https://www.pivotaltracker.com/story/show/130174367

### CODE REVIEW

File: app/src/components/Home/CoverPage.jsx [hunk 1]
```diff
@@ -172,8 +189,12 @@ class CoverPage extends Component {
             </div>
             <div className={CoverPageStyle['footer-header']}>
               <div className={CoverPageStyle['contain-ldf']}>
-                <span className={CoverPageStyle['brought-text']}>Brought to you by:</span>
-                <img className={CoverPageStyle['ldf-logo']} src={LogoLDF} alt="logo"></img>
+                <img
+                  className={classnames(CoverPageStyle['ldf-logo'], this.state.windowWidth <= 768 ? 'small-ldf' : null)}
```
Comment (clementprdhomme) [inline app/src/components/Home/CoverPage.jsx]: You could better use this syntax:
```javascript
className({
  [CoverPageStyle['ldf-logo']]: true,
  'small-ldf': this.state.windowWidth <= 768
})
```

File: app/styles/components/c-cover-page.scss [hunk 2]
```diff
@@ -222,19 +231,28 @@
       margin-top: 15px;
       width: 140px;
     }
+
+    :global .small-ldf {
```
Comment (clementprdhomme) [inline app/styles/components/c-cover-page.scss]: Do we need this style to be global?

File: app/src/components/Home/CoverPage.jsx [hunk 3]
```diff
@@ -122,6 +131,13 @@ class CoverPage extends Component {
       beforeChange: (currentSlider, nextSlider) => {
         this.onSliderChange(nextSlider);
       },
+      afterChange: this.state.windowWidth <= 768 ? () => {
```
Comment (clementprdhomme) [inline app/src/components/Home/CoverPage.jsx]: Two comments here. First, I would prefer to have the condition inside the callback than having a tertiary operator returning or a function or `null`. Second, I think you're way of determining if "Leo"' slide is active is wrong: the `afterChange` callback receives the current active slide. I would thus save in a variable the index of the slide and just check if the current one matches it.

File: app/src/components/Home/CoverPage.jsx [hunk 4]
```diff
@@ -34,12 +35,20 @@ class CoverPage extends Component {
     this.props.getCoverPageEntries();
   }
 
+  // componentWillReceiveProps() {
```
Comment (clementprdhomme) [inline app/src/components/Home/CoverPage.jsx]: If not used, just remove it. We don't want to introduce extra code.

File: app/src/components/Home/CoverPage.jsx [hunk 5]
```diff
@@ -24,7 +24,8 @@ class CoverPage extends Component {
       currentSlider: 0,
       autoPlaySlider: true,
       speedPlaySlider: 10000,
-      windowWidth: window.innerWidth
+      windowWidth: window.innerWidth,
+      leoTrue: false
```
Comment (clementprdhomme) [inline app/src/components/Home/CoverPage.jsx]: This is never used.

**Comment by clementprdhomme** [comment]
I think your PR is wrong. The task doesn't specify that the foundation's logo has to be bigger on "Leo"'s slide.

**Comment by hectoruch** [comment]
I just fix all @clementprdhomme

**Comment by clementprdhomme** [comment]
@hectoruch I removed the useless refs and squashed the commits into one so we don't have useless commits.

**Pull request merged**